app/2.2.x/admin-api.md

```diff
@@ -3475,11 +3475,7 @@ HTTP 200 OK
 
 A target is an ip address/hostname with a port that identifies an instance of a backend
 service. Every upstream can have many targets, and the targets can be
-dynamically added. Changes are effectuated on the fly.
-
-Because the upstream maintains a history of target changes, the targets cannot
-be deleted or modified. To disable a target, post a new one with `weight=0`;
-alternatively, use the `DELETE` convenience method to accomplish the same.
+dynamically added, modified or deleted. Changes are effectuated on the fly.
```

Comment:
```suggestion
dynamically added, modified, or deleted. Changes effectuate on the fly.
```

Reply:
@javierguerragiraldez The corresponding PR mentioned below used the language "takes effect" rather than "effectuate". Both are fine, and I guess this will get overwritten eventually if it isn't changed to match.

Reply:
yes, i forgot that this is a generated file; so I removed the changes here.   as for the exact wording, i guess "take effect" is the most readable version.